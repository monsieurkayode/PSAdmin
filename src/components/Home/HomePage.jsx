import React from 'react';
import { Link } from 'react-router-dom';

/**
 *
 * @function HomePage
 *
 * @returns {JSX}
 */
const HomePage = () => (
  <div className="container jumbotron">
    <div className="container-fluid">
      <h1>Pluralsight Administration</h1>
      <p>React, React Router, and Flux for ultra-responsive web apps.</p>
      <Link to="/about">
        <button type="button" className="btn btn-lg btn-primary">
          Learn more
        </button>
      </Link>
    </div>
  </div>
);

export default HomePage;
