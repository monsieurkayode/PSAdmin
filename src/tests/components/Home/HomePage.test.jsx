import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../../components/Home/HomePage';

describe('HomePage Component', () => {
  const wrapper = shallow(<HomePage />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
