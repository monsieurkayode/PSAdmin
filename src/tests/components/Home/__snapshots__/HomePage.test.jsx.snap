// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component renders correctly 1`] = `
<div
  className="container jumbotron"
>
  <div
    className="container-fluid"
  >
    <h1>
      Pluralsight Administration
    </h1>
    <p>
      React, React Router, and Flux for ultra-responsive web apps.
    </p>
    <Link
      replace={false}
      to="/about"
    >
      <button
        className="btn btn-lg btn-primary"
        type="button"
      >
        Learn more
      </button>
    </Link>
  </div>
</div>
`;
